### PR TITLE
Implement JUnit Tests for updated Edit command

### PR DESCRIPTION
**Pull request by louisdavinlie**

### CODE REVIEW

**Comment by lzf834** [comment]
Read the changes. No errors/issues. Looks good to merge.

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -36,8 +36,9 @@
     public static final String VALID_EMAIL_BOB = "bob@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
-    public static final String VALID_STATUS_AMY = "Negative";
-    public static final String VALID_STATUS_BOB = "Negative";
+    public static final String VALID_STATUS_POSITIVE = "Positive";
```
Comment (joshuayapwj98) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:39]: Just wondering if we should continue to segregate the variable to follow the convention? i.e VALID_STATUS_AMY

But do not see any issues with this! :)

**Review by joshuayapwj98 — APPROVED** [review]
LGTM!

Thanks for the fast JUnit test :)

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (louisdavinlie) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:39]: I think status is similar to tag (e.g. VALID_TAG_HUSBAND) in a sense that anyone can have the status. I feel like following the convention means only Amy/Bob can have that status. The convention makes sense for name, phone, email, and address as they are kind of unique to each person but I'm not sure if it makes sense for status (and maybe class code?) The variables are there to help test the commands, so for example if I'm changing a person's status to positive, it wouldn't make sense if I change their status to VALID_STATUS_AMY. What do you think?

**Pull request merged**